lib/full-site-editing/full-site-editing.php

```diff
@@ -143,3 +143,25 @@ function gutenberg_site_editor_load_block_editor_scripts_and_styles( $is_block_e
 		: $is_block_editor_screen;
 }
 add_filter( 'should_load_block_editor_scripts_and_styles', 'gutenberg_site_editor_load_block_editor_scripts_and_styles' );
+
+/**
+ * Used by wp_template, wp_template_part and wp_global_styles to map post type capabilities to edit_theme_options.
+ *
+ * @return array Default capabilities.
+ */
+function gutenberg_get_default_capabilities_for_fse() {
```

Comment:
I think this function is not very useful as part of our public API so we should mark it as internal by saying In the comments that it is internal and prefixing it with "_".

Regarding the name I guess it can more specific _gutenberg_get_default_capabilities_for_fse_post_types.

Reply:
Amended.